Sources/Tracking/CustomerIOImplementation+ScreenViews.swift

```diff
@@ -58,6 +59,13 @@ internal extension UIViewController {
             return
         }
         let nameOfViewControllerClass = String(describing: type(of: viewController))
+
+        // SwiftUI is a wrapper around UIKit classes. So, SwiftUI apps may have noisey screenview tracking.
+        // Filter out screenview tracking events for Views that belong to the SwiftUI framework.
+        guard !viewController.isSwiftUIView else {
```

Comment:
Do we have any instrumentation test suite that can run and validate this for us?

Reply:
I thought of that and unfortunately, no, we do not have a test suite at this time for this type of test. 

There is quite a bit of work involved to introduce this kind of test suite.[ I made a ticket](https://github.com/customerio/issues/issues/11047) for introducing this type of test suite.